front/src/router/index.js

```diff
@@ -20,6 +20,21 @@ const routes = [
         path: "/about",
         component: () => import("../views/AboutView.vue"),
       },
+      {
+        name: "artwork",
+        path: "/artwork/:id_artwork",
+        component: () => import("../views/ArtworkView.vue"),
+      },
+      {
+        name: "artwork_list",
+        path: "/artwork",
+        component: () => import("../views/Artwork_list.vue"),
```

Comment:
Make the artwork list the home page, remove the HelloWorld component and the vue logo